app/build.gradle

```diff
@@ -34,29 +33,28 @@ android {
     packagingOptions {
         exclude 'META-INF/maven/com.google.guava/guava/pom.properties'
         exclude 'META-INF/maven/com.google.guava/guava/pom.xml'
+        exclude 'META-INF/jersey-module-version'
     }
 }
 
 dependencies {
-    compile fileTree(include: ['*.jar'], dir: 'libs')
-    testCompile 'junit:junit:4.12'
-    compile 'com.android.support:appcompat-v7:25.1.0'
-
-    compile ('com.microsoft.identity.client:msal:0.1.+') {
-        exclude group: 'com.android.support', module: 'appcompat-v7'
-    }
-    compile 'com.android.volley:volley:1.0.0'
+    implementation fileTree(include: ['*.jar'], dir: 'libs')
+    testImplementation 'junit:junit:4.12'
+    implementation 'com.android.support:appcompat-v7:27.0.0'
+    implementation 'com.android.volley:volley:1.1.0'
     // Include the SDK as a dependency
-    compile 'com.microsoft.graph:msgraph-sdk-android:1.3.2'
-
+    implementation 'com.microsoft.graph:microsoft-graph:1.2.0'
+    implementation 'com.microsoft.identity.client:msal:0.2.2'
```

Comment:
This is not required since you are using android auth already

Reply:
Removed. Thanks